scico/linop/xray/astra.py

```diff
@@ -227,6 +227,85 @@ class XRayTransform3D(LinearOperator):  # pragma: no cover
     Perform tomographic projection (also called X-ray projection) of a
     volume at specified angles, using the
     `ASTRA toolbox <https://github.com/astra-toolbox/astra-toolbox>`_.
+    The `3D geometries <https://astra-toolbox.com/docs/geom3d.html#projection-geometries>`__
+    "parallel3d" and "parallel3d_vec" are supported by this interface.
+
+    The volume is fixed with respect to the coordinate system, centered
+    at the origin, as illustrated below:
+
+    .. plot:: pyfigures/xray_3d_vol.py
+       :align: center
+       :include-source: False
+       :show-source-link: False
+
+    The voxels sides have unit length (in arbitrary units), which defines
+    the scale for all other dimensions in the source-volume-detector
+    configuration. Geometry axes `x`, `y`, and `z` correspond to volume
+    array axes 0, 1, and 2 respectively. The projected array axes 0, 1,
```

Comment:
Consider remarking that this is different from astra's default interface, were I think x y z corresponds to 2 1 0.

Reply:
Good catch - fixed.